card/lib/generators/card.rb

```diff
@@ -0,0 +1,43 @@
+# -*- encoding : utf-8 -*-
```

Comment:
I will remove this if we really don't need it.  It does seem like mostly we just need to have the documentation that tells them what they need to do.  At the very least, I've agreed that it doesn't make sense to think of the step to add cards to an app be a migration generation, but it will need to run something that acts like a migration in that it adds the necessary models to db/schema.rb.

I think the rake tasks for loading the seeds should work to initialize the wagn data after adding to the schema.